forge-gui/res/cardsfolder/c/ceta_sanctuary.txt

```diff
@@ -9,6 +9,5 @@ SVar:Z:Count$Valid Permanent.Green+YouCtrl/LimitMax.1
 SVar:X:SVar$Y/Plus.Z
 SVar:C:Count$Compare X EQ2.2.1
```

Comment:
Shouldn't that C not be needed?

The Trigger (because its an Intervening IF Clause should only resolve when you get Red or Green
meaning that if you take Y and Z that each can be either 0 or 1
and Plus them with X, then it only can't be 1 or 2
because if its 0, the trigger should not be able to resolve

meaning you don't need to compare it with 2?